src/MediaStreamTrackFake.test.ts

```diff
@@ -30,6 +30,14 @@ describe('MediaStreamTrackFake', () => {
     track.stop()
     expect(track.readyState).toBe('ended')
   })
+
+  test('onEnded is called after stop', () => {
+    const onEndedListener = jest.fn()
+    track.onended = onEndedListener
+    track.stop()
+    expect(onEndedListener).toHaveBeenCalled()
```

Comment:
```suggestion
    expect(onEndedListener).not.toHaveBeenCalled()
```
This [answer](https://stackoverflow.com/a/55960232) explains why and has links to the spec.

Reply:
Thanks for the linked ticket. Addressed your comment.